ompi/request/grequest.c

```diff
@@ -203,6 +214,17 @@ int ompi_grequest_invoke_query(ompi_request_t *request,
     int rc = OMPI_SUCCESS;
     ompi_grequest_t *g = (ompi_grequest_t*) request;
 
+    /*
```

Comment:
Why would this be the case ? If we assume this is true, then I would argue we don't need the second argument to this call, because the internal status can always be accessed directly from the request itself.